docs/source/programming_resources/tutorial_specific/android_studio/disable_instant_run/disable-instant-run.rst

```diff
@@ -1,5 +1,5 @@
-Disabling Android Studio Instant Run
-====================================
+Disabling Android Studio Instant Run :bdg-warning:`Legacy` :bdg-success:`AS`
+=============================================================================
```

Comment:
Let's make headings tags equal to the content above them. I know the specification says "at least as long" but we have thus far been really good about "exactly the length".